### PR TITLE
voila-vuetify 0.1.0

### PR DESCRIPTION
**Pull request by mariobuikhuizen**
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/voila-vuetify/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ccf5258dcaa7816998a0b0db7983aaf37c181fda625780ee539765b8a39db74c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (scopatz) [inline recipes/voila-vuetify/meta.yaml]: These should be `host`  requirements.  And Python should also be a run req

File: recipes/voila-vuetify/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - test -f "${PREFIX}/share/jupyter/voila/templates/vuetify-base/nbconvert_templates/voila.tpl"
+    - test -f "${PREFIX}/share/jupyter/voila/templates/vuetify-default/conf.json"
+    - test -f "${PREFIX}/share/jupyter/voila/templates/vuetify-default/nbconvert_templates/app.html"
+
+about:
+  home: http://github.com/quantstack/voila-vuetify
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'voila-vuetify'
```
Comment (scopatz) [inline recipes/voila-vuetify/meta.yaml]: This summary should probably be expanded

**Comment by scopatz** [comment]
Thanks!

**Comment by mariobuikhuizen** [comment]
Thanks to you!

**Pull request merged**